drivers/eks/eks_driver.go

```diff
@@ -485,6 +467,67 @@ func (d *Driver) createConfigMap(state state, endpoint string, capem []byte, nod
 	return nil
 }
 
+func getClientset(state state, endpoint string, capem []byte) (*kubernetes.Clientset, error) {
+	token, err := getEKSToken(state)
+	if err != nil {
+		return nil, fmt.Errorf("error generating token: %v", err)
+	}
+
+	config := &rest.Config{
+		Host: endpoint,
+		TLSClientConfig: rest.TLSClientConfig{
+			CAData: capem,
+		},
+		BearerToken: token,
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, fmt.Errorf("error creating clientset: %v", err)
+	}
+
+	return clientset, nil
+}
+
+const awsCredentialsDirectory = "./.aws"
```

Comment:
can you change to `./management-state/aws`  everything else where we write to disk is some subfolder of management-state